hpx/runtime/actions/basic_action.hpp

```diff
@@ -807,6 +836,23 @@ namespace hpx { namespace serialization
 #define HPX_DEFINE_GET_ACTION_NAME(action)                                    \
     HPX_DEFINE_GET_ACTION_NAME_(action, action)                               \
 /**/
+#if defined(HPX_HAVE_ITTNOTIFY) && !defined(HPX_HAVE_APEX)
```

Comment:
Right, I need to re-integrate the changes from the itt_notify branch to here. Rebasing onto master will not help at this point.